.github/workflows/operator.yaml

```diff
@@ -28,13 +28,27 @@ jobs:
     steps:
       - name: Checkout Code
         uses: actions/checkout@v3
+
+      - name: Enable port-forwarding
+        run: |
+          sudo apt-get -y install socat  
```

Comment:
nitpick: is `socat` needed? the tunnel should already do the job.

Reply:
yes it's needed